concepts/return/introduction.md

```diff
@@ -0,0 +1,28 @@
+# Return
+
+In Crystal so does always the last expression in a method return its value.
```

Comment:
The introduction and about look the same, is this intentional?

Reply:
The introduction is about how the concept looks before you start the exercise and about is what it looks after. I quite often add some extra tidbit in about but other than that so are they the same